src/helpers/index.tsx

```diff
@@ -9,139 +9,166 @@ import { abi as BondDaiContract } from "../abi/bonds/DaiContract.json";
 import { abi as ReserveOhmDaiContract } from "../abi/reserves/OhmDai.json";
 import { abi as ReserveOhmFraxContract } from "../abi/reserves/OhmFrax.json";
 import { abi as FraxBondContract } from "../abi/bonds/FraxContract.json";
+import { JsonRpcSigner, StaticJsonRpcProvider } from "@ethersproject/providers";
+import { IERC20, OlympusBondDepository } from "src/typechain";
+export { default as Transactor } from "./Transactor";
 import { abi as EthBondContract } from "../abi/bonds/EthContract.json";
-
 import { SvgIcon } from "@material-ui/core";
 import { ReactComponent as OhmImg } from "../assets/tokens/token_OHM.svg";
 import { ReactComponent as DaiImg } from "../assets/tokens/DAI.svg";
 import { ReactComponent as OhmDaiImg } from "../assets/tokens/OHM-DAI.svg";
 import { ReactComponent as FraxImg } from "../assets/tokens/FRAX.svg";
 import { ReactComponent as OhmFraxImg } from "../assets/tokens/OHM-FRAX.svg";
 import { ReactComponent as wETHImg } from "../assets/tokens/wETH.svg";
-import { indexOf } from "src/contracts/YourContract.abi";
 
-export function addressForBond({ bond, networkID }) {
+export function addressForBond({ bond, networkID }: { bond: string; networkID: number }) {
+  const bondAddresses = addresses[networkID].BONDS as Nested;
   if (bond === BONDS.ohm_dai) {
-    return addresses[networkID].BONDS.OHM_DAI;
+    return bondAddresses.OHM_DAI;
   }
   if (bond === BONDS.dai) {
-    return addresses[networkID].BONDS.DAI;
+    return bondAddresses.DAI;
   }
   if (bond === BONDS.ohm_frax) {
-    return addresses[networkID].BONDS.OHM_FRAX;
+    return bondAddresses.OHM_FRAX;
   }
   if (bond === BONDS.frax) {
-    return addresses[networkID].BONDS.FRAX;
-  }
-  if (bond === BONDS.eth) {
-    return addresses[networkID].BONDS.ETH;
+    return bondAddresses.FRAX;
+  } else {
+    // bond === BONDS.eth
+    return bondAddresses.ETH;
   }
 }
 
-export function addressForAsset({ bond, networkID }) {
+export function addressForAsset({ bond, networkID }: { bond: string; networkID: number }) {
+  const reserveAddresses = addresses[networkID].RESERVES as Nested;
   if (bond === BONDS.ohm_dai) {
-    return addresses[networkID].RESERVES.OHM_DAI;
+    return reserveAddresses.OHM_DAI;
   }
   if (bond === BONDS.dai) {
-    return addresses[networkID].RESERVES.DAI;
+    return reserveAddresses.DAI;
   }
   if (bond === BONDS.ohm_frax) {
-    return addresses[networkID].RESERVES.OHM_FRAX;
+    return reserveAddresses.OHM_FRAX;
   }
   if (bond === BONDS.frax) {
-    return addresses[networkID].RESERVES.FRAX;
-  }
-  if (bond === BONDS.eth) {
-    return addresses[networkID].RESERVES.ETH;
+    return reserveAddresses.FRAX;
+  } else {
```

Comment:
why not just keep the if? (feels like a big switch case instead)

Reply:
I opted for else otherwise the function technically can return undefined and I had to force cast the function as not null everywhere it was being used otherwise.